src/assets/styles/other/_material-reduction.scss

```diff
@@ -871,6 +871,18 @@ body .mat-calendar-next-button::after {
   }
 }
 
+body .mat-vertical-content {
+  @media(max-width: $breakpoint-sm) {
+    padding: 0 8px 8px;
+  }
+}
+
+.hide-xs {
```

Comment:
I'd prefer not to add global styles and to just have it scoped locally in the component.